src/main/java/de/tum/in/www1/artemis/domain/Exercise.java

```diff
@@ -814,19 +815,19 @@ public void setTutorParticipations(Set<TutorParticipation> tutorParticipations)
         this.tutorParticipations = tutorParticipations;
     }
 
-    public Long getNumberOfParticipations() {
-        return numberOfParticipationsTransient;
+    public DueDateStat getNumberOfSubmissions() {
+        return numberOfSubmissionsTransient;
     }
 
-    public void setNumberOfParticipations(Long numberOfParticipations) {
-        this.numberOfParticipationsTransient = numberOfParticipations;
+    public void setNumberOfSubmissions(DueDateStat numberOfnumberOfSubmissions) {
+        this.numberOfSubmissionsTransient = numberOfnumberOfSubmissions;
```

Comment:
This is probably a typo? numberOfnumberOfSubmissions

Reply:
Done